### DISCUSSION TITLE
unable to setup PKI infrastructure

### DISCUSSION THREAD

**Discussion by moondersen** [cloudnative-pg/cloudnative-pg#225] [General]
Getting this error... any help would be apppreciated.

`
{"level":"info","ts":1654030838.540224,"logger":"setup","msg":"Starting CloudNativePG Operator","version":"1.15.0","build":{"Version":"1.15.0","Commit":"80cfb96f","Date":"2022-04-21"}}
{"level":"info","ts":1654030838.540261,"logger":"setup","msg":"Listening for changes on all namespaces"}
{"level":"info","ts":1654030839.0421436,"logger":"controller-runtime.metrics","msg":"Metrics server is starting to listen","addr":":8080"}
{"level":"info","ts":1654030839.0424583,"logger":"setup","msg":"Loading configuration from ConfigMap","namespace":"edb-postgres-operator","name":"postgresql-operator-controller-manager-config"}
{"level":"info","ts":1654030839.0438652,"logger":"setup","msg":"Loading configuration from Secret","namespace":"edb-postgres-operator","name":"postgresql-operator-controller-manager-config"}
{"level":"info","ts":1654030839.0455313,"logger":"setup","msg":"Operator configuration loaded","configuration":{"webhookCertDir":"","watchNamespace":"","enablePodDebugging":false,"operatorNamespace":"edb-postgres-operator","operatorPullSecretName":"cnpg-pull-secret","operatorImageName":"sx-asw-foundation-docker.art.spacex.corp/cloudnative-pg:1.0.0","postgresImageName":"ghcr.io/cloudnative-pg/postgresql:14.2","inheritedAnnotations":null,"inheritedLabels":null,"enableInstanceManagerInplaceUpdates":false,"enableAzurePVCUpdates":false,"monitoringQueriesConfigmap":"postgresql-operator-default-monitoring","monitoringQueriesSecret":""}}
{"level":"info","ts":1654030839.0472424,"logger":"setup","msg":"Kubernetes system metadata","systemUID":"81d06d75-c748-4585-9644-35c31020ea83","haveSCC":false}
{"level":"error","ts":1654030839.051571,"logger":"setup","msg":"unable to setup PKI infrastructure","error":"no deployment detected","stacktrace":"github.com/cloudnative-pg/cloudnative-pg/pkg/management/log.(*logger).Error\n\tpkg/management/log/log.go:127\ngithub.com/cloudnative-pg/cloudnative-pg/internal/cmd/manager/controller.RunController\n\tinternal/cmd/manager/controller/controller.go:203\ngithub.com/cloudnative-pg/cloudnative-pg/internal/cmd/manager/controller.NewCmd.func1\n\tinternal/cmd/manager/controller/cmd.go:35\ngithub.com/spf13/cobra.(*Command).execute\n\tpkg/mod/github.com/spf13/cobra@v1.4.0/command.go:856\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\tpkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974\ngithub.com/spf13/cobra.(*Command).Execute\n\tpkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902\nmain.main\n\tcmd/manager/main.go:61\nruntime.main\n\t/opt/hostedtoolcache/go/1.17.8/x64/src/runtime/proc.go:255"}
Error: no deployment detected`

**Comment by moondersen** [comment]
Please disregard. Bad helm config.

**Comment by sxd** [comment]
@moondersen ok! thank you!
